controllers/controllers.errors.js

```diff
@@ -0,0 +1,16 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(error);
+  }
+};
+
+exports.handle404s = (req, res, next) => {
+  res.status(404).send({ msg: "Content not found" });
```

Comment:
needs a next to 500 and if statement to determine if is a 404

Reply:
Shouldn't do, we only 404 on endpoints we dont recognise